.circleci/config.yml

```diff
@@ -313,6 +314,11 @@ jobs:
           name: hadoLint_graalvm
           command: |
             docker run --rm -i hadolint/hadolint < docker/graalvm/Dockerfile
+      - run:
+          name: Java_17
+          command: |
+            sudo apt install -q --assume-yes openjdk-17-jre-headless openjdk-17-jdk-headless
+            sudo update-java-alternatives -a
```

Comment:
does this replace an equivalent step for java11 or is this new?

Reply:
The Arm64 CircleCI "machine" image by default contains Java 11, the other CircleCI Ubuntu based machine image contains Java 17 by default. This step installs Java 17 in the Arm64 machine image and set it as default.